### PR TITLE
Adding earth-osm

### PR DESCRIPTION
**Pull request by pz-max**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pz-max** [comment]
@mnm-matin and @davide-f can you please approve that you want to be added as maintainer of `earth-osm`

**Comment by davide-f** [comment]
> @mnm-matin and @davide-f can you please approve **by adding a comment** that you want to be added as maintainer of `earth-osm`

I confirm, I would like to be a maintainer of earth-osm. Thanks max!

**Comment by mnm-matin** [comment]
confirming

**Comment by pz-max** [comment]
@conda-forge/help-python Ready for review, thanks

**Comment by pz-max** [comment]
@conda-forge-admin, please ping team

File: recipes/earth-osm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{% set name = "earth-osm" %}
+{% set version = "0.0.8" %}
+{% set sha = "b0fbd294fa4ad02df07510b0e1494fd665ffd6aa6adc7712e9dcf08844881b75" %}
```
Comment (ocefpaf) [inline recipes/earth-osm/meta.yaml:3]:
```suggestion
```

File: recipes/earth-osm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/earth_osm-{{ version }}.tar.gz
+  sha256: {{ sha }}
```
Comment (ocefpaf) [inline recipes/earth-osm/meta.yaml:11]: No need for a jinja variable that it used only once.

```suggestion
  sha256: b0fbd294fa4ad02df07510b0e1494fd665ffd6aa6adc7712e9dcf08844881b75
```

**Pull request merged**